express/code/blocks/pricing-table/pricing-table.css

```diff
@@ -868,4 +929,40 @@ main .section .pricing-table-wrapper:has(.pricing-table.many-cols) {
   .pricing-table.seamless .toggle-content{
     display: none !important;
   }
-}
+
+  .pricing-table.pricing-table.seamless .row.table-start-row:not(.connect-to-toggle) .col:last-child  {
+    border-right: none;
+  }
+
+  .pricing-table.pricing-table.seamless .row.table-start-row:not(.connect-to-toggle) .col:first-child  {
+    border-left: none;
+  }
+
+  .pricing-table.pricing-table.seamless .row.table-end-row:not(.connect-to-toggle) .col:last-child {
+    border-bottom-right-radius:0px;
+    border-bottom-left-radius:0px;
+  }
+
+  .pricing-table.pricing-table.seamless .row.table-end-row:not(.connect-to-toggle) .col:first-child {
```

Comment:
This selector could be combined with line 941 since they are both setting `border-bottom-right-radius` and `border-bottom-left-radius` to 0.